akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -381,8 +430,15 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
 
   val (afrFileChannel, afrFlie, flightRecorder) = initializeFlightRecorder()
 
+  def createFlightRecorderEventSink(): EventSink = {
+    // FIXME there is some concurrency issue with the FlightRecorder, when shutting down.
+    //       It crashes the JVM.
+    // flightRecorder.createEventSink()
+    IgnoreEventSink
```

Comment:
@drewhk my jvm started to crash with these changes to the shutdown. Probably some threading issue. 

from the hs_err_pid file:
```
Stack: [0x00007000039b7000,0x0000700003bb7000],  sp=0x0000700003bb5b40,  free space=2042k
Native frames: (J=compiled Java code, j=interpreted, Vv=VM code, C=native code)
J 24529 C1 sun.misc.Unsafe.getAndAddLong(Ljava/lang/Object;JJ)J (27 bytes) @ 0x0000000110804360 [0x0000000110804300+0x60]
j  org.agrona.concurrent.MappedResizeableBuffer.getAndAddLong(JJ)J+24
j  akka.remote.artery.RollingEventLogSection.writeRecord$1(Ljava/nio/ByteBuffer;Lorg/agrona/concurrent/MappedResizeableBuffer;)V+14
j  akka.remote.artery.RollingEventLogSection.write(ILjava/nio/ByteBuffer;)V+10
j  akka.remote.artery.FlightRecorder$$anon$1.loFreq(I[B)V+35
j  akka.remote.artery.AeronSource$$anon$1.postStop()V+48
j  akka.stream.impl.fusing.GraphInterpreter.finalizeStage(Lakka/stream/stage/GraphStageLogic;)V+1
j  akka.stream.impl.fusing.GraphInterpreter.finish()V+28
j  akka.stream.impl.fusing.GraphInterpreterShell.tryAbort(Ljava/lang/Throwable;)V+84
j  akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$postStop$1.apply(Lakka/stream/impl/fusing/GraphInterpreterShell;)V+5
j  akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$postStop$1.apply(Ljava/lang/Object;)Ljava/lang/Object;+5
J 20567 C1 scala.collection.immutable.Set$Set1.foreach(Lscala/Function1;)V (12 bytes) @ 0x000000011042d394 [0x000000011042d280+0x114]
j  akka.stream.impl.fusing.ActorGraphInterpreter.postStop()V+25
```